app/components/works/contributor_row_component.html.erb

```diff
@@ -1,7 +1,9 @@
 <div class="contributor row inner-container">
   <div class="col-md-12">
-    <%= button_tag "🗑", class: 'btn btn-sm float-right', aria: { label: 'Remove' },
-        data: { action: "click->nested-form#remove_association" } %>
+    <%= button_tag class: 'btn btn-sm float-right', aria: { label: 'Remove' },
+        data: { action: "click->nested-form#remove_association" } do %>
+      <span class="far fa-trash-alt"></span>
```

Comment:
Does the `far` class mean we are using FontAwesome pro? https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use seems to suggest it. (I didn't know we had pro!) Although looking at this icon, it says it's in the regular set but it's also free? https://fontawesome.com/icons/trash-alt?style=regular I'm confused. (But this PR is A-OK.)

Reply:
I think the s in "fas" may stand for "solid" and the r in "far" may stand for regular.  I got the info from here:

![image](https://user-images.githubusercontent.com/96775/98139432-0fce8180-1e79-11eb-921f-b9c6cbf6e9e4.png)

Reply:
![image](https://user-images.githubusercontent.com/96775/98139536-2c6ab980-1e79-11eb-8c94-998c8ea3e337.png)

Reply:
I think you're right about what r and s mean. I didn't get the impression that the regular icons were deprecated. But the FA documentation on whether or not the regular icons are free or pro or both twisted my wig real good. Not important, though. What's important: the icon matches what's in the design doc!